src/components/SignupForm/index.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+import logo from '../../assets/logo.png';
+import ButtonForm from '../ButtonForm';
+
+import './SignupForm.scss';
+
+function SignupForm() {
+  const { register, handleSubmit, errors, getValues } = useForm();
+
+  const isPasswordEqual = (password: string) => password === getValues('password');
+
+  const onSubmit = handleSubmit(({ firstName, lastName, email, password, confirmPassword }) => {
+    const USER = {
+      firstName,
+      lastName,
+      email,
+      password,
+      confirmPassword
+    };
+    console.log({ user: USER });
+  });
+
+  return (
+    <div className="signup-form">
+      <img src={logo} className="logo" />
+      <form onSubmit={onSubmit}>
+        <div>
+          <label>Nombre</label>
+          <span className="error">{errors.firstName && errors.firstName.message}</span>
+          <input name="firstName" ref={register({ required: 'Requerido' })} />
```

Comment:
I suggest you move the validators to a utilities file, so that in future forms you can reuse them and make use of them again